nixos/modules/services/networking/dhcpcd.nix

```diff
@@ -207,13 +207,19 @@ in
 
         serviceConfig =
           { Type = "forking";
-            PIDFile = "/run/dhcpcd.pid";
+            PIDFile = "/run/dhcpcd/pid";
```

Comment:
Is this the upstream default? I would expect it to be like this

```suggestion
            PIDFile = "/run/dhcpcd/dhcpcd.pid";
```

Reply:
yes :/ though technically the upstream default is `/var/run/dhpcd/pid`, this elides that `/var/run -> /run` link

Reply:
Does dhcpd manage this directory?

It seems we should also explicitly manage this with RuntimeDirectory, that would prevent situations where dhcpd might forget to clean up after itself.

Reply:
it does, but setting RuntimeDirectory does make sense anyway.